models/parent.ts

```diff
@@ -0,0 +1,14 @@
+import type { provinces, locales } from "@prisma/client";
+
+export type CreateParentInput = {
+    first_name: string;
+    last_name: string;
+    phone_number: string;
+    is_low_income?: boolean | null;
```

Comment:
do you need to add the `null` because the question mark indicates the field is optional?

Reply:
@jason17h